package.json

```diff
@@ -85,8 +83,10 @@
     "rimraf": "^2.5.4",
     "source-map-support": "^0.4.11",
     "sourceify": "^0.1.0",
-    "uglify-js": "^2.8.26",
-    "watchify": "^3.11.0"
+    "uglify-es": "^3.3.9",
+    "watchify": "^3.11.0",
+    "webpack": "^4.28.3",
```

Comment:
Does this package actually use Webpack? It's added here, but it doesn't seem it's used before or after your changes.

Reply:
ah, no it doesn't but I forgot to remove it.